File: app/src/main/java/com/shuai/retrofitrx/example/app/ui/bean/CheckRomBean.java
package com.shuai.retrofitrx.example.app.ui.bean;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 @author changshuai
 */

public class CheckRomBean implements Serializable {

    @SerializedName("result")
    private String result;

    public String getResult() {
        return result;
    }

    public void setResult(String result) {
        this.result = result;
    }

    @Override
    public String toString() {
        return "CheckRomBean{" +
                "result='" + result + '\'' +
                '}';
    }
}

File: app/src/main/java/com/shuai/retrofitrx/example/app/ui/bean/CheckRomBean.kt
package com.shuai.retrofitrx.example.app.ui.bean

import java.io.Serializable

/**
 * @author changshuai
 */
class CheckRomBean : Serializable {
    var result: String = ""
    override fun toString(): String {
        return "CheckRomBean(result='$result')"
    }

}